### ISSUE TITLE
A survey on deep learning based face recognition

### ISSUE THREAD

**Issue by odauk001** [Yawna/Convolutional-Neural-Networks#19]
### Title
A survey on deep learning based face recognition
### URL
https://www.sciencedirect.com/science/article/pii/S1077314219301183?casa_token=TSiNA-qVXU0AAAAA:bErP12vUGbfHPtDhnlaId46PK9a73N8Hc4AZJlIOJPixjOcDIVJCAULwKjzpVU4eIsDwjG5eVoig
### Summary 
This is a survey that talks about deep learning, specifically deep convolutional neural networks, based facial recognition methods. Within the deep learning methods it talks about CNNs, single CNN, and multi-CNN. Within single CNN it talks about how methods fuse different types of face features, how they utilize metric learning algorithms, loss functions, and activation functions. Within multi-CNN it talks about extracting features in certain regions of the face and extracting those features. It also goes into variants of CNNs, talking about their layouts, ways of learning kernels, fusing CNNs with other modules, and weakly supervised and unsupervised learning. 
### Citation
Guodong Guo, Na Zhang,
A survey on deep learning based face recognition,
Computer Vision and Image Understanding,
Volume 189,
2019,
102805,
ISSN 1077-3142,
https://doi.org/10.1016/j.cviu.2019.102805.
(https://www.sciencedirect.com/science/article/pii/S1077314219301183)
Abstract: Deep learning, in particular the deep convolutional neural networks, has received increasing interests in face recognition recently, and a number of deep learning methods have been proposed. This paper summarizes about 330 contributions in this area. It reviews major deep learning concepts pertinent to face image analysis and face recognition, and provides a concise overview of studies on specific face recognition problems, such as handling variations in pose, age, illumination, expression, and heterogeneous face matching. A summary of databases used for deep face recognition is given as well. Finally, some open challenges and directions are discussed for future research.
Keywords: Deep learning; Face recognition; Artificial Neural Network; Convolutional Neural Networks; Autoencoder; Generative Adversarial Networks
